box-ui/src/translations/en.json

```
@@ -27,7 +27,12 @@
         "selectCamera": "Select camera ",
         "camera": "camera",
         "joinAMeeting": "Join a meeting",
-        "linkForTeacher": "I am a teacher",
+        "linkForTeacher": "To teacher space",
+        "linkForStudent": "To student space",
+        "teacherSpace": "Teacher Space",
+        "studentSpace": "Student Space",
+        "teacher": "Teacher",
+        "student": "Élève",
```

```suggestion
        "student": "Student",
```
English section here ^^